blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -139,9 +139,18 @@ export const LeadArtPresentation = (props) => {
 							width={800}
 							height={450}
 							responsiveImages={[800, 1600]}
-							resizedOptions={{ auth: lead_art.auth[RESIZER_APP_VERSION], smart: true }}
+							resizedOptions={{ auth: lead_art.auth[2], smart: true }}
 							resizerURL={RESIZER_URL}
```

Comment:
If you pass `ansImage={lead_art}` to the `Image`, you don't need the auth token, url, or the resizer url and can remove this import (noted above)

You'd have to modify the other image too (`galleryItem`) to remove this dependency

Reply:
I know this is not your doing, but `lead_art` should be `leadArt` to keep with our naming conventions.

Reply:
Also, considering I recommend removing this so it doesn't matter, but we should never be hardcoding the `RESIZER_APP_VERSION`

Reply:
@nschubach Thanks. Fixed this [bcfd60a](https://github.com/WPMedia/arc-themes-blocks/pull/1588/commits/bcfd60ad133d39765c9978ede17115524966fc72)